sections/futures/MobileTrade/UserTabs/UserTabs.tsx

```diff
@@ -48,16 +48,23 @@ const UserTabs: React.FC = () => {
 };
 
 const UserTabsContainer = styled.div`
-	padding: 0 15px;
 	min-height: 390px;
 `;
 
 const TabButtonsContainer = styled.div`
 	display: grid;
 	grid-template-columns: repeat(4, 1fr);
-	grid-column-gap: 15px;
 	margin-bottom: 15px;
 	overflow: auto;
+
+	> button {
```

Comment:
Noticed this is a repeat of above, could we break it out to a re-usable component, or add it to the TabButton itself?

Reply:
Done in latest commit.